src/components/admin/form-components/withSelector.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import update from 'react-addons-update';
+import _ from 'lodash';
+
+import { capFirstLetter } from 'lib/utilities';
+import { getDisplayName } from 'lib/higher-order-helpers';
+
+// material-ui
+import Chip from 'material-ui/Chip';
+
+// HOCs
+import { withModals } from 'components/admin/hocs/modals/withModals';
+
+export const withSelector = WrappedField => {
```

Comment:
Could we maybe make this argument a bit clearer so it's purpose is clear. What is a good description of it's purpose? `Searcher`? `Finder`? `SearchBar`?

Reply:
Finder it is :)